InstallESD_Hashes.csv

```diff
@@ -13,3 +13,4 @@ Version,Build,SHA-256,SHA-1
 10.12.1,16B2659,8efa85e12bcc6c2145cce68b6ecaf9ce23e11f58c1452982b1907fe0f9f76fd1,f7f147c54627c2a9beb1fa318394e1579b30b167
 10.12.2,16C67,6c2b16f248407a3853a9c4a63efadc94813321708f5eed5c09b73f33e5edd855,1432e3be6222c434b536721076ed8b16b1c6050e
 10.12.2,16C68,6e8ccda1849bb49b1acf75f455019fe327adb47c676dbff018ea811c2456dcce,94f9e8f7ae2540dee6fe3465f60fc037e2547d16
+10.12.3,16D32,75a288fe6efc0591f757baf08305270f1b843b54cfb66fe6b257049400a0d6e9,77d354ec06df0d0acc37c105ae524ba96948142b
```

Comment:
[I attest, this pair of checksum is correct.](http://macintoshgarden.org/apps/mac-os-x-intel-microprocessor)